third_party/txt/src/minikin/FontCollection.cpp

```diff
@@ -528,8 +530,8 @@ void FontCollection::itemize(const uint16_t* string,
           }
           start -= prevChLength;
         }
-        result->push_back(
-            {family->getClosestMatch(style), static_cast<int>(start), 0});
+        result->push_back({family->getClosestMatchWithChar(style, ch),
```

Comment:
Shouldn't we also use variant to check glyphs?
```C++
result = hb_font_get_glyph(hb_font, codepoint, variant, &unusedGlyph);
```

Reply:
I didn't know about ``variant selector``, so I looked it up. I think you're right. :smile: